### PR TITLE
PHP 8.4 support added

### PR DESCRIPTION
**Pull request by Atul-glo35265**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Comment by Atul-glo35265** [comment]
@froschdesign @gsteel,
How can we fix the Autocloser / autoclose (pull_request) failure?

**Comment by gsteel** [comment]
The autocloser is supposed to automatically close new PRs and issues to this repo because it is no longer actively maintained except for PHP upgrades - the workflow itself is broken. Just ignore it!

File: .laminas-ci.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "ignore_php_platform_requirements": {
-        "8.3": true
+        "8.3": true,
```
Comment (gsteel) [inline .laminas-ci.json]: This line is no longer required

**Review by gsteel — CHANGES_REQUESTED** [review]

File: .laminas-ci.json (hunk 1 above)
Comment (Atul-glo35265) [reply .laminas-ci.json]: Done

**Review by gsteel — APPROVED** [review]
Thanks @Atul-glo35265

**Pull request merged**